src/Common/MemoryWorker.cpp

```diff
@@ -118,7 +132,7 @@ struct CgroupsV2Reader : ICgroupsReader
     {
         std::lock_guard lock(mutex);
         stat_buf.rewind();
-        return readMetricFromStatFile(stat_buf, "anon");
+        return readMetricsFromStatFile(stat_buf, {"anon", "sock"});
```

Comment:
can you extract this change in a different PR so it's merged ASAP?

Reply:
https://github.com/ClickHouse/ClickHouse/pull/74747 (after a bit more research added `kernel` too)